package.json

```diff
@@ -50,6 +50,7 @@
     "@tailwindcss/forms": "^0.5.7",
     "@tailwindcss/line-clamp": "^0.4.4",
     "@tailwindcss/typography": "^0.5.10",
+    "@types/react-syntax-highlighter": "^15.5.11",
```

Comment:
With the latest PR [merged](https://github.com/asyncapi/website/pull/2734/files#diff-ab8f79b68b7adff7a07db953bf453f3c5aa6ade98d2b1b67d8432b36392489edR13), you can now move this to `devDependencies`.

Reply:
Done